### PR TITLE
Added Utkarsh1504 #361

### LINKED ISSUE #361: Utkarsh Mishra
this is a screenshot of my profile.

### PR DESCRIPTION
**Pull request by Utkarsh1504**
resolve #361

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 [
+  
+  {
```
Comment (AllanRegush) [inline data.json:3]: Please add yourself to the bottom

**Review by AllanRegush — COMMENTED** [review]
Thanks for the PR :+1: Looking Good :+1:

File: data.json (hunk 1 above)
Comment (Utkarsh1504) [reply data.json]: done

File: data.json [hunk 2]
```diff
@@ -599,4 +599,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/29686102/106919215-f0711a00-672f-11eb-88f6-8ab44f67b911.png",
     "issueNumber": 313
   }
+  {
```
Comment (eddiejaoude) [inline data.json:602]: This needs a `,` between the objects

File: data.json [hunk 3]
```diff
@@ -599,4 +599,10 @@
...
+    "name": "Utkarsh Mishra",
+    "githubUsername": "Utkarsh1504",
+    "imageUrl": "https://user-images.githubusercontent.com/76392681/107881996-f673b180-6f0c-11eb-90a8-9365ff3e4753.png",
+    "issueNumber": 361
+},
```
Comment (eddiejaoude) [inline data.json:607]: - Indentation is not quite right 
- Last object does not need a `,`

Thank you for the contribution, just a few small tweaks, then we can get it merged in 👍

Comment (Utkarsh1504) [reply data.json]: Thank you for your guidance. Now I have fixed the indentation.

Comment (eddiejaoude) [reply data.json]: Awesome, thank you 👍

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**